docs/networks/staking/03-schedule.md

```diff
@@ -34,8 +34,8 @@ See [Epoch Preparation Protocol](./04-epoch-preparation.md#epoch-length) for det
 
 </Callout>
 
-**Staking Operations are disabled for approximately the last day of an epoch**,
-typically around 12:00pm (7:00pm UTC) on Tuesday every week until the next day around the same time.
+**Staking Operations are disabled for approximately the last 6-12 hours of an epoch**,
+typically around 12:00am US Pacific Time (7:00am UTC) on Wednesday every week until around 12:00pm US Pacific Time (7:00pm UTC).
```

Comment:
```suggestion
typically around 00:00 US Pacific Daylight Time (07:00 UTC) on Wednesday every week until around 12:00 US Pacific Daylight Time (19:00 UTC).
```

Midnight is pretty confusing to represent in 12hr time. We could use 24hr (suggested). I know we use 12hr in many other places though, so to keep it consistent we could also keep 12hr but add a minute to make it more obvious: 12:01AM. Things like transit schedules often do this to make the meaning more obvious at a glance.